bangc-ops/kernels/psroipool/psroipool.cpp

```diff
@@ -181,3 +265,69 @@ mluOpStatus_t MLUOP_WIN_API mluOpPsRoiPoolForward(
   GEN_CASE_END();
   return MLUOP_STATUS_SUCCESS;
 }
+
+mluOpStatus_t MLUOP_WIN_API mluOpPsRoiPoolBackward(
+    mluOpHandle_t handle, const int pooled_height, const int pooled_width,
+    const float spatial_scale, const int output_dim,
+    const mluOpTensorDescriptor_t top_grad_desc, const void *top_grad,
+    const mluOpTensorDescriptor_t rois_desc, const void *rois,
+    const mluOpTensorDescriptor_t mapping_channel_desc,
+    const void *mapping_channel, const mluOpTensorDescriptor_t bottom_grad_desc,
+    void *bottom_grad) {
+  const std::string api = "[mluOpPsRoiPoolBackward]";
+  mluOpStatus_t ret = psRoiPoolBackwardParamCheck(
+      api, handle, pooled_height, pooled_width, spatial_scale, output_dim,
+      top_grad, rois, mapping_channel, bottom_grad, top_grad_desc, rois_desc,
+      mapping_channel_desc, bottom_grad_desc);
+  if (ret != MLUOP_STATUS_SUCCESS) {
+    LOG(ERROR) << api
+               << " Error found during element verification, please check.";
+    return MLUOP_STATUS_BAD_PARAM;
```

Comment:
MLUOP_STATUS_NOT_SUPPORTED 没考虑到

```suggestion
    return ret;
```

Reply:
已修改